# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/binutils/have_stdlib.patch
# ROCK Linux is Copyright (C) 1998 - 2005 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./libiberty/cplus-dem.c.orig	Thu Nov  8 01:11:59 2001
+++ ./libiberty/cplus-dem.c	Thu Nov  8 01:14:09 2001
@@ -48,12 +48,7 @@
 #pragma weak dlerror
 #endif
 
-#ifdef HAVE_STDLIB_H
 #include <stdlib.h>
-#else
-char * malloc ();
-char * realloc ();
-#endif
 
 #include <demangle.h>
 #undef CURRENT_DEMANGLING_STYLE
